### PR TITLE
Compute Identity support

### PR DESCRIPTION
**Pull request by prankulmahajan**
Test results : https://github.ibm.com/alchemy-containers/armada-storage/issues/3722

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,62 +3,81 @@ module github.com/IBM/ibmcloud-volume-file-vpc
 go 1.18
 
 require (
-	github.com/BurntSushi/toml v0.3.1
 	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7
-	github.com/IBM/ibm-csi-common v1.0.0-beta17
-	github.com/IBM/ibmcloud-volume-interface v1.0.0-beta10
+	github.com/IBM/ibm-csi-common v1.0.0-beta21
+	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta7.0.20221014111644-b1f895203a78
+	github.com/IBM/secret-common-lib v1.0.4-0.20221017105240-21e98d568724
```
Comment (GunaKKIBM) [inline go.mod:9]: Created a release for secret-common-lib too, please update the same.

Comment (prankulmahajan) [reply go.mod:9]: Sure. Will update both secret-common-lib and interface lib once https://github.com/IBM/ibmcloud-volume-interface/pull/25 is merged

**Comment by prankulmahajan** [comment]
Tested with latest commit. 
https://github.ibm.com/alchemy-containers/armada-storage/issues/3722#issuecomment-49676238

**Review by GunaKKIBM — APPROVED** [review]
lgtm

**Review by arahamad — APPROVED** [review]
lgtm, verify changes as well

**Pull request merged**